content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -0,0 +1,161 @@
+---
+reviewers:
+- klueska
+- pohly
+title: Dynamic Resource Allocation
+content_type: concept
+weight: 65
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+Dynamic resource allocation is a new API for requesting and sharing resources
```

Comment:
I think we should mention here that this API is conceptually similar to the persistent volumes API. For those familiar with the persistent volumes API, they will immediately have a mental model to work off of when reading the rest of the documentation.